cmd/traffic/cmd/manager/internal/mutator/watcher.go

```diff
@@ -230,6 +231,15 @@ type configWatcher struct {
 	data       map[string]map[string]string
 	modCh      chan entry
 	delCh      chan entry
+
+	configUpdatersMapLock sync.RWMutex
```

Comment:
My preference would be to name this `configUpdatersLock` to correspond with the attribute name of what it lockes. Mixing in the type in that attribute in the name is just confusing.